s3/settings.py

```diff
@@ -0,0 +1,29 @@
+"""
+Configuration for the S3 Interface microservice settings.
+"""
+import json
+import logging
+import os
+
+log = logging.getLogger(__name__)
+
+
+def get_s3_info():
+    vcap_services = os.getenv("VCAP_SERVICES")
+    try:
+        credentials = json.loads(vcap_services)["s3"][0]["credentials"]
+    except (json.JSONDecodeError, KeyError) as err:
+        log.warning("Unable to load info from VCAP_SERVICES")
+        log.debug("Error: %s", str(err))
+
+    key_id = credentials["access_key_id"]
```

Comment:
wouldn't this fall through from the except scope to the value lookup? Perhapps a re-throw after printing error to avoid another keyerror. Not sure what the idiomatic way to do this is.